### PR TITLE
doc(release): prepare release notes for Modules 23.04

### PR DESCRIPTION
**Pull request by pkippes**
"map 23.04.14
it edition ext 23.04.3
mbi 23.04.6
bam 23.04.7"

## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [x] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.mdx [hunk 1]
```diff
@@ -760,7 +795,18 @@ Release date: `April 26, 2023`
...
 
-### 23.104.2
+### 23.04.3
+
+Release date: `March 12, 2024`
+
+<details>
+  <summary>Bug fixes</summary>
+
+- When the license is not valid, expired, renamed, or moved, users now get an error page instead of multiple error messages.
```
Comment (jdelpierre) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.mdx:805]: I think it's not the correct fix.

it's concerns : 
The IT Edition module does not generate a license error snackbar, and the custom page is not displayed if the license is invalid.

File: versioned_docs/version-23.04/releases/centreon-commercial-extensions.mdx [hunk 2]
```diff
@@ -758,6 +793,17 @@ Release date: `April 26, 2023`
...
 ## Centreon IT Edition Extensions
 
+### 23.04.3
+
+Release date: `March 12, 2024`
+
+<details>
+  <summary>Bug fixes</summary>
+
+- When the license is not valid, expired, renamed, or moved, users now get an error page instead of multiple error messages.
```
Comment (jdelpierre) [inline versioned_docs/version-23.04/releases/centreon-commercial-extensions.mdx:803]: I think it's not the correct fix.

it's concerns :
The IT Edition module does not generate a license error snackbar, and the custom page is not displayed if the license is invalid.

Comment (cg-tw) [reply versioned_docs/version-23.04/releases/centreon-commercial-extensions.mdx:803]: ok, what would be a correct message?

Comment (smau-centreon) [reply versioned_docs/version-23.04/releases/centreon-commercial-extensions.mdx:803]: As discussed with @lpinsivy we should say `Removed the error that was displayed when a license was missing or invalid.`
OK for you?

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.mdx [hunk 3]
```diff
@@ -21,6 +21,29 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
+
+- [Homepage] The loading page has been improved.
+
+</details>
+
+<details>
+  <summary>Bug fixes</summary>
+
+- [Geoviews] Fixed an issue that prevented the selection of more than 10 host groups.
+- [Homepage] Removed improper label in MAP homepage.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.mdx:39]:
```suggestion
- [Homepage] Removed incorrect label in MAP homepage.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.mdx [hunk 4]
```diff
@@ -21,6 +21,29 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
+- [Viewer] Added "acknowledged" and "in downtime" icons on the list of host services.
+- [Viewer] Fixed widget values and resource status updates when compression is activated.
+- [Viewer] Fixed the Business Activity name display in the tooltip.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.mdx:42]:
```suggestion
- [Viewer] Fixed the Business Activity name displayed in the tooltip.
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.mdx (hunk 1 above)
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-commercial-extensions.mdx:805]:
```suggestion
- Removed the error that was displayed when a license was missing or invalid.
```

File: versioned_docs/version-23.04/releases/centreon-commercial-extensions.mdx (hunk 2 above)
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-commercial-extensions.mdx:803]:
```suggestion
- Removed the error that was displayed when a license was missing or invalid.
```

File: versioned_docs/version-23.04/releases/centreon-commercial-extensions.mdx [hunk 5]
```diff
@@ -22,6 +22,29 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
...
+
+- [Homepage] The loading page has been improved.
+
+</details>
+
+<details>
+  <summary>Bug fixes</summary>
+
+- [Geoviews] Fixed an issue that prevented the selection of more than 10 host groups.
+- [Homepage] Removed improper label in MAP homepage.
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-commercial-extensions.mdx:40]:
```suggestion
- [Homepage] Removed incorrect label in MAP homepage.
```

File: versioned_docs/version-23.04/releases/centreon-commercial-extensions.mdx [hunk 6]
```diff
@@ -22,6 +22,29 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
...
+- [Viewer] Added "acknowledged" and "in downtime" icons on the list of host services.
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-commercial-extensions.mdx:41]:
```suggestion
- [Viewer] Added "acknowledged" and "in downtime" icons to the list of host services.
```

File: versioned_docs/version-23.04/releases/centreon-commercial-extensions.mdx [hunk 7]
```diff
@@ -22,6 +22,29 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
...
+- [Viewer] Fixed widget values and resource status updates when compression is activated.
+- [Viewer] Fixed the Business Activity name display in the tooltip.
```
Comment (cg-tw) [inline versioned_docs/version-23.04/releases/centreon-commercial-extensions.mdx:43]:
```suggestion
- [Viewer] Fixed the Business Activity name displayed in the tooltip.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by rsadik-ext — APPROVED** [review]

**Review by rsadik-ext — APPROVED** [review]

**Review by smau-centreon — APPROVED** [review]

**Pull request merged**